### PR TITLE
Add an option to log container properties

### PR DESCRIPTION
**Pull request by garyrussell**
https://stackoverflow.com/questions/47914528/how-to-print-spring-kafka-configurations-in-logs-using-log4j/47914975#comment82800055_47914975

### CODE REVIEW

**Comment by artembilan** [comment]
Travis is in a panic after this change 😄

**Comment by garyrussell** [comment]
Yeah - fixed

File: spring-kafka/src/main/java/org/springframework/kafka/listener/config/ContainerProperties.java [hunk 1]
```diff
@@ -483,4 +487,55 @@ public void setClientId(String clientId) {
 		this.clientId = clientId;
 	}
 
+	/**
+	 * Log the container configuration if true (INFO).
+	 * @return true to log.
+	 * @since 2.0.1
```
Comment (artembilan) [inline spring-kafka/src/main/java/org/springframework/kafka/listener/config/ContainerProperties.java]: Are you really going to backport it?

File: spring-kafka/src/test/java/org/springframework/kafka/listener/ConcurrentMessageListenerContainerTests.java [hunk 2]
```diff
@@ -102,6 +102,7 @@ public void testAutoCommit() throws Exception {
 		Map<String, Object> props = KafkaTestUtils.consumerProps("test1", "true", embeddedKafka);
 		DefaultKafkaConsumerFactory<Integer, String> cf = new DefaultKafkaConsumerFactory<>(props);
 		ContainerProperties containerProps = new ContainerProperties(topic1);
+		containerProps.setLogContainerConfig(true);
```
Comment (artembilan) [inline spring-kafka/src/test/java/org/springframework/kafka/listener/ConcurrentMessageListenerContainerTests.java]: Need to document this one.
Thanks

File: spring-kafka/src/main/java/org/springframework/kafka/listener/config/ContainerProperties.java (hunk 1 above)
Comment (garyrussell) [reply spring-kafka/src/main/java/org/springframework/kafka/listener/config/ContainerProperties.java]: oops, I keep forgetting master is 2.1

**Pull request merged**